.devcontainer/docker-compose.yml

```diff
@@ -34,24 +34,49 @@ services:
 
     # define environment variables
     environment:
+      INTERNAL_FILE_REGISTRY_SERVICE_CONFIG_YAML: /workspace/.devcontainer/.dev_config.yaml
       DB_URL: postgresql://postgres:postgres@internal_file_registry_service_db/postgres
```

Comment:
Should we harmonize these URLs across all microservices?

Reply:
Yes, I have that on my list and will add a new PR to the template repo in the coming days.